tests/test_simplelog.py

```diff
@@ -0,0 +1,192 @@
+import sys
+
+import re
+import click
+import logging
+
+log_stream = None
+
+
+def tool_resetlog():
```

Comment:
I think, that capturing the stdout and stderr into result.output doesn't work reliably. At least not when running the tests via pytest.

Maybe I could rewrite the code to simply check against result.output, commit it and you could test it on your side?